lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :data do
+  desc 'Update resources with nil quantity'
```

Comment:
This is not very accurate, as we are not really doing that (updating the resources that have nil quantity).
It is also not very descriptive, as it doesn't say how we are updating them.